protocol/protocol.tex

```diff
@@ -2738,14 +2745,54 @@
 \ascii{zt}.
 }
 
+\nsubsubsection{Viewing Keys} \label{viewingkeyencoding}
+
+\changed{
+A \viewingKey consists of $\AuthPublic$ and $\TransmitPrivate$.
+$\AuthPublic$ is a $\SHAName$ output.
+$\TransmitPrivate$ is a $\KAPrivate$ key (see \crossref{concretekeyagreement}),
+for use with the encryption scheme defined in \crossref{inband}. These
+components are derived from a \spendingKey as described in \crossref{keycomponents}.
+
+\introlist
+The raw encoding of a \viewingKey consists of:
+\vspace{2ex}
+\begin{equation*}
+\begin{bytefield}[bitwidth=0.07em]{520}
+    \bitbox{80}{$8$-bit $\ViewingKeyLeadByte$}
+    \bitbox{80}{$8$-bit $\ViewingKeySecondByte$}
+    \bitbox{256}{$256$-bit $\AuthPublic$}
+    \bitbox{256}{\changed{$256$}-bit $\TransmitPrivate$}
+\end{bytefield}
+\end{equation*}
+
+\begin{itemize}
+    \item Two bytes $[\ViewingKeyLeadByte, \ViewingKeySecondByte]$,
+        indicating this version of the raw encoding of a \Zcash \viewingKey
+        on the production network. (Addresses on the test network use
+        $[\ViewingKeyTestnetLeadByte, \ViewingKeyTestnetSecondByte]$
+        instead.)
+    \item 256 bits specifying $\AuthPublic$.
+    \item 256 bits specifying $\TransmitPrivate$, using the normal encoding
+        of a Curve25519 private key \cite{Bern2006}.
```

Comment:
That's a good point. Should we reject keys that are not clamped? That seems the simplest way to avoid divergence.